### ISSUE TITLE
Create figma pages for v1 of website/mobile

### ISSUE THREAD

**Issue by ebele-oputa** [hackforla/open-community-survey#54]
### Dependency
- #17 
- And the issues that are created from 17 (are in new issue approval, waiting on the urls this issue will provide).  When those are made, Kalyani and Bonnie will reorg the figma.
  - #76 
  - #73
  - #116
  - #117
  - #118
  - #119
  - #112
  - #113
  - #114
  - #115

### Overview
We need to create separate Figma pages for each website page and one page with all the designs, so that you can play the website and each design team member can work on a webpage without effecting the workspace of another designer.

### Action Items
- [ ] Create pages
- [ ] Move versions to the pages they belong to
- [ ] clean up full site page
- [ ] add dates last updated
- [ ] define where next versions go

### Resources/Instructions
[Website Figma](https://www.figma.com/file/04T31AqsprIIkuBN4OfwOo/Open-Community-Survey-Designs?node-id=0%3A1)

#### Structure
pages:
full site (updated date)
  - Home
  - About
  - Get Involved
  - Partners
  - Projects
    - NC website analysis
    - empowerla.org redesign
